stylelint.config.js

```diff
@@ -1,289 +1,295 @@
 // Transform from PascalCase to kebab-case
-const pascalToKebabCase = str => str.replace(/\.?([A-Z])/g, (char) =>  "-" + char.toLowerCase()).replace(/^-/, "");
+const pascalToKebabCase = (str) => str.replace(/\.?([A-Z])/g, (char) => '-' + char.toLowerCase()).replace(/^-/, '');
 // Given a component name in PascalCase, returns a regex. The regex
 // must match CSS selectors conforming to the BEM naming conventions
 // you want to enforce.
-const customBemSelector = component => {
+const customBemSelector = (component) => {
   const block = pascalToKebabCase(component);
-  const kebabCase = "[a-z#{}$]+(?:-[a-zA-Z0-9#{}$]+)*";
+  const kebabCase = '[a-z#{}$]+(?:-[a-zA-Z0-9#{}$]+)*';
   const element = `(?:__${kebabCase})?`;
   const modifier = `(?:--${kebabCase})?`;
-  const attribute = "(?:\\[.+\\])?";
+  const attribute = '(?:\\[.+\\])?';
   return new RegExp(`^\\.${block}${element}${modifier}${attribute}$`);
 };
 
 module.exports = {
   plugins: [
-    "stylelint-order", 
-    "stylelint-selector-bem-pattern",
-    "stylelint-scss",
-    "stylelint-declaration-use-variable",
+    'stylelint-order',
+    'stylelint-selector-bem-pattern',
+    'stylelint-scss',
+    'stylelint-declaration-use-variable'
   ],
   rules: {
-    "scss/media-feature-value-dollar-variable": "always",
-    "sh-waqar/declaration-use-variable": [["color", "background-color", { ignoreValues: ["inherit"] }]],
-    "plugin/selector-bem-pattern": {
-      preset: "bem",
-      implicitComponents: "src/components/**/*.scss",
+    'scss/media-feature-value-dollar-variable': 'always',
+    'sh-waqar/declaration-use-variable': [ [ 'color', 'background-color', { ignoreValues: [ 'inherit' ] } ] ],
+    'plugin/selector-bem-pattern': {
+      preset: 'bem',
+      implicitComponents: 'src/components/**/*.scss',
       componentSelectors: {
         initial: customBemSelector
       }
     },
-    "order/order": [
+    'order/order': [
       [
-				"dollar-variables",
-				"custom-properties",
-				"at-rules",
-				"declarations",
-				{
-					type: "at-rule",
-					name: "media",
+        'dollar-variables',
+        'custom-properties',
+        'at-rules',
+        'declarations',
+        {
+          type: 'at-rule',
+          name: 'media'
```

Comment:
mike pushed up changes to this file (there will be conflicts)

Reply:
I'll be ready